lib/active_model/serializer.rb

```diff
@@ -27,12 +27,12 @@ class Serializer
       )
     /x
 
-    class << self
-      attr_accessor :_attributes
-      attr_accessor :_attributes_keys
-    end
-
     with_options instance_writer: false, instance_reader: false do |serializer|
+      class_attribute :_type, instance_reader: true
+      class_attribute :_attributes
+      self._attributes ||= []
+      class_attribute :_attributes_keys
```

Comment:
well, I guess it's my personal preference to group that kind of stuff together. makes it slightly easier to skim, imo